Package.swift

```diff
@@ -14,22 +14,52 @@ import PackageDescription
 let package = Package(
   name: "SwiftProtobuf",
   products: [
-    .executable(name: "protoc-gen-swift", targets: ["protoc-gen-swift"]),
-    .library(name: "SwiftProtobuf", targets: ["SwiftProtobuf"]),
-    .library(name: "SwiftProtobufPluginLibrary", targets: ["SwiftProtobufPluginLibrary"]),
+    .executable(
+        name: "protoc-gen-swift",
+        targets: ["protoc-gen-swift"]
+    ),
+    .library(
+        name: "SwiftProtobuf",
+        targets: ["SwiftProtobuf"]
+    ),
+    .library(
+        name: "SwiftProtobufPluginLibrary",
+        targets: ["SwiftProtobufPluginLibrary"]
+    ),
+    .plugin(
+        name: "SwiftProtobufPlugin",
+        targets: ["SwiftProtobufPlugin"]
+    ),
   ],
   targets: [
-    .target(name: "SwiftProtobuf"),
-    .target(name: "SwiftProtobufPluginLibrary",
-            dependencies: ["SwiftProtobuf"]),
-    .target(name: "protoc-gen-swift",
-            dependencies: ["SwiftProtobufPluginLibrary", "SwiftProtobuf"]),
-    .target(name: "Conformance",
-            dependencies: ["SwiftProtobuf"]),
-    .testTarget(name: "SwiftProtobufTests",
-                dependencies: ["SwiftProtobuf"]),
-    .testTarget(name: "SwiftProtobufPluginLibraryTests",
-                dependencies: ["SwiftProtobufPluginLibrary"]),
+    .target(
+        name: "SwiftProtobuf"
+    ),
+    .target(
+        name: "SwiftProtobufPluginLibrary",
+        dependencies: ["SwiftProtobuf"]
+    ),
+    .executableTarget(
+        name: "protoc-gen-swift",
+        dependencies: ["SwiftProtobufPluginLibrary", "SwiftProtobuf"]
+    ),
+    .target(
+        name: "Conformance",
+        dependencies: ["SwiftProtobuf"]
+    ),
+    .plugin(
```

Comment:
Apart from the reformatting, it looks like you've added `.executableTarget` and `.plugin` entries.

Reply:
I haven't really added a new `executableTarget`. The `protoc-gen-swift` target was always executable but with newer tools versions we introduced `executableTarget`. This change here is just fixing this warning up.